vue-frontend/src/components/android-app-development/DevelopmentSection.vue

```diff
@@ -18,7 +15,78 @@
         growth and customer engagement.</span
       >
     </div>
-    <div class="swiper-content tw-mt-8">
+    <!-- Desktop UI -->
+    <div class="sticky-parent tw-hidden md:tw-block">
```

Comment:
this space is too much for 4k devices
![Screenshot from 2023-09-08 12-29-52](https://github.com/canopas/canopas-website/assets/85920585/36c7293d-d66c-4d3e-9dac-512035e23c66)

Reply:
its requires as scroll will not work

Reply:
but it is not looking good we can't take this much of space

Reply:
lets discuss this space thing with sir.